package.json

```diff
@@ -19,6 +19,7 @@
     "test": "jest",
     "seed": "medusa seed -f ./data/seed.json",
     "start": "yarn build && medusa start",
+    "dev": "yarn build && medusa develop",
```

Comment:
suggestion: just a thought, not everybody have yarn install, but everybody have nm with node. should we change yarn to npm in the scripts, I ve seen users having issues because it was not installed

Reply:
Good point! I'll make that change